src/core/runner.vala

```diff
@@ -0,0 +1,13 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private interface Games.Runner : Object {
+	public abstract Gtk.Widget? get_display ();
```

Comment:
> I think embedding the game widget inside the app is also out of focus.
> I wouldn't expect GNOME Documents to replace LibreOffice, for example.

> Can you remind me what was the point in embedding the widget?
> Was it for the snapshot/preview feature?

Imagine a music player where to play any music, an external application is launched and you have different applications for every music type. It's bad an that's kind of what we already have for games, because before libretro we had no better way to do so. Now we can treat machine emulators and some game engines as codecs!

Of course GNOME Games' goal is not to deprecate ScummVM of ZSNES and their bazillions of fine tuning, just as GNOME Music don't replace a MIDI music player, but we can offer simple and tight integration for thousands of games now, and no application on Linux does that! They can be presented and handled in the same way, we can imagine very high level features like snapshoting... having the widget inside the app isn't out of focus, it's precisely the focus of the app! I would say that its the non tighly integrated games which are out of focus in fact. =)